lib/ansible/module_utils/vmware.py

```diff
@@ -66,6 +66,22 @@ def find_obj(content, vimtype, name, first=True):
     return [obj for obj in obj_list if obj.name == name]
 
 
+def find_objs(content, vimtypes, container=None, name=None):
+    """ Get all objects of certain type(s) and optionally match name """
+    items = []
+    cv = content.viewManager.CreateContainerView(container or content.rootFolder,
+                                                 vimtypes, recursive=True)
+    # make sure name does not contain a path
+    if name:
+        name = name.split('/')[-1]
```

Comment:
I think this is not required, name may not contain `/`

Reply:
That's what I expected but without it the integration tests failed.
```yaml
# test/integration/targets/vmware_guest/tasks/cdrom_d1_c1_f0.yml (line 38):
cluster: "{{ clusterlist['json'][0] }}"
# translates into
cluster: "/dc0/host/dc0_c0"
```
So this part can be removed if the test is updated.
```yaml
cluster: "{{ clusterlist['json'][0]|basename }}"
```
Although it doesn't hurt to keep it I think.